doc/source/admin/security.md

```diff
@@ -0,0 +1,21 @@
+# Security considerations
+
+### Protect Galaxy against data loss due to misbehaving tools
+
+Tools have access to the paths of input and output data sets which are stored in
+``file_path`` and by default the credentials used for running tools are the same
+as for running Galaxy. Thus its possible that tools modify data in Galaxy's
+``file_path``. Examples for such changes are:
+
+- Creation of additional files, e.g. indices, which is a problem for cleaning up data, because Galaxy does not know about these files.
+- Removal of input or output files of the tools. This will create problems with other tools using these data sets (note that most tool repositories use CI tests to to avoid this, but the problem may still occur).
```

Comment:
```suggestion
- Removal of tool input or output files. This will create problems with other tools using these datasets (note that most tool repositories use CI tests to to avoid this, but the problem may still occur).
```